recipes/_configure_server.rb

```diff
@@ -193,7 +209,6 @@
     }
     allotted_pool_size += pool_max_size
   end
-  node.run_state['cerner_splunk'] ||= {}
```

Comment:
Good catch. I thought I added it back.
a130c949abf8c44e857f763abbc931851068ea9d